### ISSUE TITLE
Error in output

### ISSUE THREAD

**Issue by amiruak05** [gradio-app/gradio#2910]
### Describe the bug

I don't know what is happenning to my code, It doesnt predict the output,
 I have finished training the model successfully,
but gradio is giving me headache, it shows  Error(in the output box) when  i input an image,  see my code below:

!pip install gradio

import gradio as gr
import tensorflow as tf
import numpy as np
import requests

inception_net = tf.keras.applications.InceptionV3()
response = requests.get("https://drive.google.com/file/d/1jsX7SbCuu-XGMBtX-GqMC-hfNyt62lTY/view?usp=share_link")
labels = response.text.split("\n")
def classify_image(img):
  img_new = img.reshape((-1, 299, 299, 3))
  img_resize = img_new
  img = tf.keras.applications.inception_v3.preprocess_input(img)
  prediction = inception_net.predict(img).flatten()
  return {label[i]: float(prediction[i]) for i in range(2)}

image = gr.inputs.Image()
label = gr.outputs.Label(num_top_classes= 2)

my_project = gr.Interface(fn=classify_image, inputs=image, outputs=label)

my_project.launch(share = True)

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

def classify_image(img):
  img_new = img.reshape((-1, 299, 299, 3))
  img_resize = img_new
  img = tf.keras.applications.inception_v3.preprocess_input(img)
  prediction = inception_net.predict(img).flatten()
  return {label[i]: float(prediction[i]) for i in range(2)}

image = gr.inputs.Image()
label = gr.outputs.Label(num_top_classes= 2)

my_project = gr.Interface(fn=classify_image, inputs=image, outputs=label)

my_project.launch(share = True)

### Screenshot

![rtyyu](https://user-images.githubusercontent.com/109295719/210070098-77ac9460-ede8-4515-b58b-5b7559953055.PNG)

### Logs

```shell
used
```

### System Info

```shell
browser
```

### Severity

annoying

**Comment by abidlabs** [comment]
Hi @amiruak05 thanks for asking your question. However, for general questions, please use our Discord or GitHub Discussions. Only use GitHub issues if you find a bug in the Gradio codebase or if you have a feature request. 

That being said, I looked into your code, and there are several issues with it:
* The `labels` are not properly downloaded (check the `labels` variable)
* `img_new` is defined but not used

The only gradio-specific issue is that for the input, you should set the input shape to be (299, 299). Here is a working Colab with your code (it mostly works, but I'm not sure how you intend to use your labels): https://colab.research.google.com/drive/1ks1DppkHEuXKEnjUxvxLRcNnsBSwZUeI?usp=sharing
